the only failure is not related.
```
[93/138] Building C object zephyr/drivers/adc/CMakeFiles/drivers__adc.dir/adc_shell.c.obj
FAILED: zephyr/drivers/adc/CMakeFiles/drivers__adc.dir/adc_shell.c.obj 
ccache /opt/sdk/zephyr-sdk-0.11.2/arm-zephyr-eabi/bin/arm-zephyr-eabi-gcc -DBOARD_FLASH_SIZE=CONFIG_FLASH_SIZE -DBUILD_VERSION=zephyr-v2.2.0-2428-gf9d88ae88475 -DCPU_MK64FN1M0VLL12 -DKERNEL -DNDEBUG -D_FORTIFY_SOURCE=2 -D__PROGRAM_START -D__ZEPHYR_SUPERVISOR__ -D__ZEPHYR__=1 -I../../../../../../include -Izephyr/include/generated -I../../../../../../soc/arm/nxp_kinetis/k6x -I../../../../../../lib/fnmatch/. -I/home/buildslave/src/github.com/zephyrproject-rtos/modules/hal/cmsis/CMSIS/Core/Include -I/home/buildslave/src/github.com/zephyrproject-rtos/modules/hal/nxp/mcux/devices/MK64F12 -I/home/buildslave/src/github.com/zephyrproject-rtos/modules/hal/nxp/mcux/drivers/kinetis/. -isystem ../../../../../../lib/libc/minimal/include -isystem /opt/sdk/zephyr-sdk-0.11.2/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/9.2.0/include -isystem /opt/sdk/zephyr-sdk-0.11.2/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/9.2.0/include-fixed -Os -imacros/home/buildslave/src/github.com/zephyrproject-rtos/zephyr/sanity-out/frdm_k64f/tests/boards/board_shell/boards.shell.shell/zephyr/include/generated/autoconf.h -ffreestanding -fno-common -g -mcpu=cortex-m4 -mthumb -mabi=aapcs -imacros/home/buildslave/src/github.com/zephyrproject-rtos/zephyr/include/toolchain/zephyr_stdint.h -Wall -Wformat -Wformat-security -Wno-format-zero-length -Wno-main -Wno-address-of-packed-member -Wno-pointer-sign -Wpointer-arith -Wno-unused-but-set-variable -Werror=implicit-int -Werror -fno-asynchronous-unwind-tables -fno-pie -fno-pic -fno-strict-overflow -fno-reorder-functions -fno-defer-pop -fmacro-prefix-map=/home/buildslave/src/github.com/zephyrproject-rtos/zephyr/tests/boards/board_shell=CMAKE_SOURCE_DIR -fmacro-prefix-map=/home/buildslave/src/github.com/zephyrproject-rtos/zephyr=ZEPHYR_BASE -fmacro-prefix-map=/home/buildslave/src/github.com/zephyrproject-rtos=WEST_TOPDIR -ffunction-sections -fdata-sections -std=c99 -nostdinc -MD -MT zephyr/drivers/adc/CMakeFiles/drivers__adc.dir/adc_shell.c.obj -MF zephyr/drivers/adc/CMakeFiles/drivers__adc.dir/adc_shell.c.obj.d -o zephyr/drivers/adc/CMakeFiles/drivers__adc.dir/adc_shell.c.obj   -c /home/buildslave/src/github.com/zephyrproject-rtos/zephyr/drivers/adc/adc_shell.c
In file included from /home/buildslave/src/github.com/zephyrproject-rtos/zephyr/drivers/adc/adc_shell.c:7:
/home/buildslave/src/github.com/zephyrproject-rtos/zephyr/drivers/adc/adc_shell.c:477:22: error: expected expression before ',' token
  477 |  ADC_SHELL_COMMAND(0),
      |                      ^
../../../../../../include/shell/shell.h:219:3: note: in definition of macro 'SHELL_STATIC_SUBCMD_SET_CREATE'
  219 |   __VA_ARGS__      \
      |   ^~~~~~~~~~~
```

--- a/west.yml
+++ b/west.yml
@@ -117,7 +117,7 @@ manifest:
       revision: 0aefdda69d236cb01f932117c1f32e9da09c0ec3
     - name: mipi-sys-t
       path: modules/debug/mipi-sys-t
-      revision: baf51863f19f009b92e762115ba5572a5b996b92
+      revision: 957d46bc3ce0d5f628f0d525196bb4db207672ee
     - name: nrf_hw_models
       path: modules/bsim_hw_models/nrf_hw_models
       revision: fec69703cb1ca06fcdab6d5fde01274f0fc5c759

